posix.mak

```diff
@@ -229,6 +229,10 @@ $(ROOT)/threadasm.o : src/core/threadasm.S
 	@mkdir -p $(dir $@)
 	$(CC) -c $(CFLAGS) $< -o$@
 
+$(ROOT)/gc/%.o : src/gc/%.c
+	@mkdir -p $(dir $@)
+	$(CC) -c $(CFLAGS) $< -o$@
```

Comment:
Why do we need this? (I don't see any C files that have been added)

Reply:
This has remained from the initial version that used C files instead of pragma(crt_constructor). I'll remove it.